src/ec/packages/ec-component-message/message.story.js

```diff
@@ -15,6 +19,40 @@ import dataWarning from './demo/data--warning';
 import message from './ecl-message.html.twig';
 import notes from './README.md';
 
+const iconsList = {};
+iconsList.none = null;
+
+notifIcons.forEach(icon => {
+  iconsList[icon] = icon;
+});
+
+const PrepareMessage = data => {
+  const name = select(
+    'icon.name',
+    iconsList,
+    data.icon.name,
+    buttonLabels.required
+  );
+  if (name !== null) {
+    getIconKnobs(data, name, 'notifications', 'l');
+  }
+
+  if (data.title) {
+    data.title = text('Title', data.title, buttonLabels.required);
+  }
+
+  if (data.description) {
```

Comment:
same here, it's mandatory, we set a knob for it, no need to check for it to be present